### PR TITLE
Fix warning in TeleinfoBase.cpp

### PR DESCRIPTION
**Pull request by mce35**

### CODE REVIEW

**Comment by gizmocuz** [comment]
Thank you for the quick fix, would this be the same as writing it

`bool bContactState = (teleinfo.STGE & 0x01) ? true : false;`

**Comment by mce35** [comment]
Yes, fixed

**Comment by gizmocuz** [comment]
Thank you again!!

File: hardware/TeleinfoBase.cpp [hunk 1]
```diff
@@ -398,7 +398,7 @@ void CTeleinfoBase::ProcessTeleinfo(const std::string& name, int rank, Teleinfo&
 	{ // Process status register if found (Linky standard mode only, refer to Enedis specs for details)
 		if((teleinfo.STGE & 0x1) != (teleinfo.prevSTGE & 0x1) || teleinfo.prevSTGE == UINT32_MAX)
 		{
-			bool bContactState = (teleinfo.STGE & 0x01 != 0)?true:false;
+			bool bContactState = (teleinfo.STGE & 0x01) ? true : false;
```
Comment (keryell) [inline hardware/TeleinfoBase.cpp:401]: Then why not
```c++
bool bContactState = teleinfo.STGE & 0x01;
```
?

**Pull request merged**